core/src/main/scala/org/apache/spark/ui/jobs/JobProgressListener.scala

```diff
@@ -374,28 +374,30 @@ class JobProgressListener(conf: SparkConf) extends SparkListener with Logging {
       execSummary.taskTime += info.duration
       stageData.numActiveTasks -= 1
 
-      val (errorMessage, metrics): (Option[String], Option[TaskMetrics]) =
+      val (errorMessage, metrics): (Option[String], Seq[AccumulableInfo]) =
```

Comment:
@andrewor14 I am not sure If I understand your comment. Can you please elaborate?